internal/terraform/lang/resource_block.go

```diff
@@ -125,8 +131,9 @@ func resourceCandidates(resources []schema.Resource) []CompletionCandidate {
 	candidates := []CompletionCandidate{}
 	for _, r := range resources {
 		candidates = append(candidates, &labelCandidate{
-			label:  r.Name,
-			detail: r.Provider,
+			label:         r.Name,
+			detail:        fmt.Sprintf("Resource, %s provider", r.Provider),
```

Comment:
Very subjective (so feel free to ignore 😄 ), but I'm thinking `"Resource (%s)", r.Provider` would provide same/similar value and reduce the length. Maybe the length is not a concern for most providers and IDEs, I'm not sure 🤷

Reply:
```suggestion
			detail:        fmt.Sprintf("Resource (%s)", r.Provider),
```